modules/administration_manual/pages/configuration/server/legal_settings_configuration.adoc

```diff
@@ -1,6 +1,13 @@
 = Legal Settings Configuration
+:experimental:
+:release-notes-imprint-and-privacy-url: https://doc.owncloud.com/server/administration_manual/release_notes.html#new-options-to-display-imprint-and-privacy-policy
+
+Because of one or more legal frameworks around the world, some ownCloud instances may need to display links to both an Imprint as well as a Privacy Policy on all pages (both in the Web UI and within email templates).
```

Comment:
Important detail: Imprint and Privacy Policy are shown on all **public** pages and in the footer of e-mails. Authenticated pages like files app or settings do not show them.